src/containers/Cluster/ClusterInfo/ClusterInfo.tsx

```diff
@@ -29,24 +32,66 @@ export const ClusterInfo = ({
 
     const clusterLinks = useClusterLinks();
 
-    const clusterInfo = getInfo(cluster ?? {}, info, [...links, ...clusterLinks]);
-
-    const getContent = () => {
-        if (loading) {
-            return <InfoViewerSkeleton className={b('skeleton')} rows={9} />;
-        }
+    const clusterInfo = getInfo(cluster ?? {}, info);
 
+    const renderInfo = () => {
         if (error && !cluster) {
             return null;
         }
 
-        return <InfoViewer dots={true} info={clusterInfo} />;
+        return (
+            <div>
+                <div className={b('section-title')}>{i18n('title_info')}</div>
+                <DefinitionList nameMaxWidth={200}>
+                    {clusterInfo.map(({label, value}) => {
+                        return (
+                            <DefinitionList.Item key={label} name={label}>
+                                {value}
+                            </DefinitionList.Item>
+                        );
+                    })}
+                </DefinitionList>
+            </div>
+        );
+    };
+
+    const renderLinks = () => {
```

Comment:
`const linksList = links.concat(clusterLinks)` ?
And farther logic should become leaner.